.circleci/config.yml

```diff
@@ -356,6 +364,7 @@ workflows:
   default-workflow:
     jobs:
       - build
+      - build-docker-image
```

Comment:
we only need this for the release jobs (?) so I guess:
```suggestion
      - build-docker-image:
          filters:
            branches:
              ignore: /.*/
```

Reply:
no, that's the contrary in fact.

Reply:
oh, right